includes/UserMailer.php

```diff
@@ -849,16 +863,16 @@ function sendMails() {
 	 * @return Boolean
 	 * @private
 	 */
-	function sendPersonalised( $watchingUser ) {
+	private function sendPersonalised( $watchingUser ) {
 		global $wgContLang, $wgEnotifUseRealName;
 		// From the PHP manual:
 		//     Note:  The to parameter cannot be an address in the form of "Something <someone@example.com>".
 		//     The mail command will not parse this properly while talking with the MTA.
 		$to = new MailAddress( $watchingUser );
 
-		# $PAGEEDITDATE is the time and date of the page change
-		# expressed in terms of individual local time of the notification
-		# recipient, i.e. watching user
+		// $PAGEEDITDATE is the time and date of the page change
+		// expressed in terms of individual local time of the notification
+		// recipient, i.e. watching user
 		$body = str_replace(
```

Comment:
That would be nice. I think I've hit my limit when it comes to refactoring all this bad code though, how about we leave that for the next time we touch this file? The rest of this class is at least 2000% better than it was before.